google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -62,6 +62,10 @@ def insert_violations(self, violations, resource_name,
             # Create the violations snapshot table.
             snapshot_table = self._create_snapshot_table(
                 resource_name, snapshot_timestamp)
+        except MySQLdb.OperationalError, e:
```

Comment:
With the way the code was, creating the violations table was done by all the scanners.  So, this is the easiest way to work around that.  I can put a TODO to check for the violations table outside of each scanners.